app/views/placements/support/schools/placements/add_placement/_check_your_answers_step.html.erb

```diff
@@ -0,0 +1,62 @@
+<%= content_for :page_title, sanitize(t(".page_title")) %>
+
+<%= form_for(check_your_answers_step, url: current_step_path, method: :put) do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l">
+        <%= t(".add_placement") %>
+      </span>
+
+      <span class="govuk-heading-l">
+        <%= t(".title") %>
+      </span>
+
+      <%= govuk_summary_list do |summary_list| %>
```

Comment:
It's probably worth making this a partial. Then any changes to the "check your answers" page of a placements are reflected for both support and school users.

Reply:
A partial specifically for this `govuk_summary_list` may not be necessary if we can reuse _all_ the step partials wholesale.